src/fromager/packagesettings.py

```diff
@@ -252,8 +252,10 @@ class BuildOptions(pydantic.BaseModel):
     build_ext_parallel: bool = False
     """Configure `build_ext[parallel]` in `DIST_EXTRA_CONFIG`
 
-    This enables parallel builds of setuptools extensions. Incompatible
-    with some packages, e.g. numba 0.60.0.
+    .. deprecated:: 1.0
```

Comment:
The version number should be the next release.

```suggestion
    .. deprecated:: 0.72.0
```